docs/strategies/creating_a_strategy.md

```diff
@@ -66,6 +66,70 @@ If you find out in the check function that you want to give a new advice to the
     this.advice('short');
     // or
     this.advice('long');
+    
+### candle variables when creating your strategy
+
+You can use different candle variables when writing your strategies, depending on what you have in mind, using additional variables might help you to improve your results.
+
+Possible candle variables are:
+ - candle.close: the closing price of the candle
+ - candle.high: the highest price of the candle
+ - candle.low: the lowest price of the candle
+ - candle.volume: the trading volume of that candle
+ - candle.trades: number of trades in that candle
+ 
+Keep in mind that this variables will give you different results depending on the candle-size you're using.
+
+
+### basic strategy example
+
+This a basic strategy example that buys and sells BTC/USDT when it hits a specific price.
+
+    // Let's create our own buy and sell strategy 
+    var strat = {};
+
+    // Prepare everything our strat needs
+    strat.init = function() {
+      // setting buy price
+      this.buyPrice = 2000;
+      
+      // setting sell price
+      this.sellPrice = 2500;
+    }
+
+    // What happens on every new candle?
+    strat.update = function(candle) {
+      // your code!
+    }
+
+    // For debugging purposes.
+    strat.log = function() {
+      // your code!
+    }
+
+    // Based on the newly calculated
+    // information, check if we should
+    // update or not.
+    strat.check = function() {
```

Comment:
this function is passed the candle, if you don't specify it as an argument you cannot access `candle.close` on the next line. Could you add that?